pkg/notifiers/slack/build_message.go

```diff
@@ -111,7 +111,7 @@ func BuildRequestMessage(o RequestMessageOpts) (summary string, msg slack.Messag
 
 	if o.WasReviewed || o.Request.Status == access.CANCELLED {
 		t := time.Now()
```

Comment:
This already existed, shouldn't this be the time that the request was review, you can get this from the event input.
Rather than the time the event was processed (now())